charts/kong/values.yaml

```diff
@@ -81,6 +81,11 @@ deployment:
 # is set below. In general, you should not set values here if they are set elsewhere.
 env:
   database: "off"
+  # the chart uses the traditional router (for Kong 3.x+) because the ingress
+  # controller generates traditional routes. if you do not use the controller,
+  # you may set this to "traditional_compatible" or "expression" to use the new
+  # DSL-based router
+  router_flavor: "traditional"
```

Comment:
Is there a doc in GW docs that we could link to here where the user could explore this option in detail at the source?

Reply:
There's https://docs.konghq.com/gateway/latest/key-concepts/routes/expressions/ but it's not really useful for explaining it in the controller context. We probably want something that covers the new router from our end, but there aren't 2.6 docs yet that we can link to from the chart comments.

Reply:
Added a task in https://github.com/Kong/charts/pull/666 depending on how much we know about the docs site status for 2.6 docs.